src/ReleaseHistory.md

```diff
@@ -3,6 +3,7 @@
 ## **v2.1.18** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.18) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.18) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.18) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.18)
 * FEATURE: Enhanced property bag serialization unit testing. [#1673](https://github.com/microsoft/sarif-sdk/issues/1673)
 * BUGFIX: Fix packaging warning NU5048 during build. [#1687](https://github.com/microsoft/sarif-sdk/issues/1687)
+* BUGFIX: Improve Result Matching sort and comparison.
```

Comment:
>. [](start = 53, length = 1)

Consider filing an issue and citing it here (see line above).

Reply:
Hmm. I tried to, but realized it's three different bugs and I don't want to write them all and cite them all. =)

Reply:
I've called out the three changes instead via comments, so at least the Release Notes describe the specific improvements.